react/components/Input/index.js

```diff
@@ -86,6 +86,7 @@ class Input extends Component {
       suffix: suffixProp,
       suffixIcon,
       button,
+      isLoadingButton,
```

Comment:
I'd change to `isLoading` since it's a pattern commonly used in other components.

Reply:
@kevinch I'm afraid of using `isLoading` on the `Input` component and looking like `Input` status instead of a `Button` status.

Reply:
Just `loading` wouldn't be enough? 🤔

Reply:
or `loading` yes. I'll add a note for the next iteration of the Styleguide to set a pattern for this kind o variable name.

Reply:
i opened a thread about it: https://vtex.slack.com/archives/C1MCRG7CK/p1571774967090800

Reply:
BUT we can think about a loading state for the `Input` component itself as well :). later 😬

Reply:
Sounds like it's a loading state for the whole component since i can see on the gif the input turning "disabled".

Reply:
Sounds like it's a loading state for the whole component since i can see on the gif the input turning "disabled".

Reply:
@kevinch it's just an optical illusion.. the input and the button have the same bg color.